website/scripts/setup.sh

```diff
@@ -1,7 +1,28 @@
 #!/bin/bash
 
-brew list hugo || brew install hugo
-which wget || brew install wget
+source ../scripts/detect_os_type.sh
+
+PLAT_FORM=`platform`
```

Comment:
@caofangkun - recommend using $PLATFORM rather than $PLAT_FORM - easier to read?